src/apps/chat/migrations/0001_squashed_0004_remove_room_category_room_category.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 4.0.8 on 2023-05-05 14:52
```

Comment:
You need to pull actual migrations.

Please don't change existing migration. 

By squashing I mean just deleting all your new migrations and creating one new from scratch.